--- a/drivers/mfd/ac200.c
+++ b/drivers/mfd/ac200.c
@@ -111,15 +111,13 @@
 	return ret;
 }
 
-static int ac200_i2c_remove(struct i2c_client *i2c)
+static void ac200_i2c_remove(struct i2c_client *i2c)
 {
 	struct ac200_dev *ac200 = i2c_get_clientdata(i2c);
 
 	regmap_write(ac200->regmap, AC200_SYS_CONTROL, 0);
 
 	clk_disable_unprepare(ac200->clk);
-
-	return 0;
 }
 
 static const struct i2c_device_id ac200_ids[] = {
