--- a/drivers/net/wireless/xradio/ap.c	2022-06-20 06:00:02.568099273 +0200
+++ b/drivers/net/wireless/xradio/ap.c	2022-06-20 06:03:44.797053291 +0200
@@ -467,9 +467,9 @@
 				/* TODO:COMBO:Change this once
 				* mac80211 changes are available */
 				BUG_ON(!hw_priv->channel);
-				hw_priv->ht_oper.ht_cap = sta->ht_cap;
-				priv->bss_params.operationalRateSet =__cpu_to_le32(
-				  xradio_rate_mask_to_wsm(hw_priv, sta->supp_rates[hw_priv->channel->band]));
+//				hw_priv->ht_oper.ht_cap = sta->ht_cap;
+//				priv->bss_params.operationalRateSet =__cpu_to_le32(
+//				  xradio_rate_mask_to_wsm(hw_priv, sta->supp_rates[hw_priv->channel->band]));
 				/* TODO by Icenowy: I think this may lead to some problems. */
 //				hw_priv->ht_oper.channel_type   = info->channel_type;
 				hw_priv->ht_oper.operation_mode = info->ht_operation_mode;
